web/src/components/secrets/SecretList.vue

```diff
@@ -2,6 +2,28 @@
   <div class="space-y-4 text-color">
     <ListItem v-for="secret in secrets" :key="secret.id" class="items-center">
       <span>{{ secret.name }}</span>
+
```

Comment:
the ui should hide this option if WOODPECKER_SECRET_ALLOW_SHOW_VALUE = false

Reply:
`secret.value` is empty when the the feature is not enabled.

Reply:
a v-if=secret.value would just hide the span